ui/src/message_popup/summary_card.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import Divider from 'material-ui/Divider';
+import VelocityTransitionGroup from 'velocity-react/velocity-transition-group';
+
+
+export default React.createClass({
+  displayName: 'SummaryCard',
+  
+  getInitialState(){
+    return ({
+      timer: this.props.timer
+    });
+  },
+  
+  render(){
+    
+    return(
+      <div>
+        <div style={styles.title}>Summary:</div>
+        <Divider />
+        <div style={styles.summary}>
+          <div style={styles.questionTitle}>Question:</div>
+          <div style={styles.questionText}>{this.props.question.text}</div>
+          <Divider />
+          <div style={styles.responseTitle}>Your Response:</div>
+          <div style={styles.responseText}>"{this.props.response}"</div>
+          <div style={styles.responseTime}>Time to respond: {this.state.timer} {this.state.timer === 1 ? "second" : "seconds"}</div>
+        </div>
+        <Divider />
+        <div style={styles.buttonRow}>
+          <RaisedButton
+            onTouchTap={this.props.onDone}
+            style={styles.button}
+            secondary={true}
+            label={this.props.isLastQuestion ? "Finish" : "Next Question"}/>
```

Comment:
It looks like the only thing this is used for is determining the button text, so could the caller pass that in directly?  Also, let's add this to the `propTypes` for this component too.